lib/api/src/modules/stories.ts

```diff
@@ -78,8 +78,8 @@ export interface SubAPI {
     parameterName?: ParameterName
   ) => Story['parameters'] | any;
   getCurrentParameter<S>(parameterName?: ParameterName): S;
-  updateStoryArgs(story: Story, newArgs: Args): void;
-  resetStoryArgs: (story: Story, argNames?: string[]) => void;
+  updateStoryArgs(story: Story | { id: StoryId; refId?: string }, newArgs: Args): void;
+  resetStoryArgs: (story: Story | { id: StoryId; refId?: string }, argNames?: string[]) => void;
```

Comment:
Shall we name this type, like `StoryKey` or something? @ndelangen did you ever give this thing a name?

Reply:
No I don't think I have a type/interface for this type of object yet.
`StoryKey` sounds good to me!

Reply:
Thanks for reviewing! Where would be a good place for me to create the `StoryKey` type? Is `lib/api/src/lib/stories.ts` a good place?

Reply:
@Ang-YC that sounds right, thanks!